test/transitions.html

```diff
@@ -0,0 +1,105 @@
+<!doctype html>
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>paper-drawer-panel tests</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../../web-component-tester/browser.js"></script>
+  <script src="../../test-fixture/test-fixture-mocha.js"></script>
+
+  <link rel="import" href="../../test-fixture/test-fixture.html">
+  <link rel="import" href="../paper-drawer-panel.html">
+</head>
+<style>
+  body {
+    margin: 0;
+    padding: 0;
+  }
+</style>
+
+<style is="custom-style">
+  #paperDrawerPanel {
+    --paper-drawer-panel-timing-function: linear;
+  }
+</style>
+<body>
+
+  <test-fixture id="drawer">
+    <template>
+        <paper-drawer-panel>
+          <div drawer></div>
+          <div main></div>
+        </paper-drawer-panel>
+    </template>
+  </test-fixture>
+
+  <test-fixture id="custom-timing-function-drawer">
+    <template>
+      <paper-drawer-panel id="paperDrawerPanel" force-narrow>
+        <div drawer></div>
+        <div main></div>
+      </paper-drawer-panel>
+    </template>
+  </test-fixture>
+
+  <script>
+    suite('transitions', function() {
+
+      test('drawer defaults to ease-in-out transition', function(done) {
+        var f, drawer;
+        f = fixture('drawer');
+        drawer = f.querySelector('#drawer');
+
+        Polymer.Base.async(function() {
+          var drawerStyle = window.getComputedStyle(drawer);
+
+          try {
+            expect(drawerStyle['transition-timing-function']).to.be.equal('ease-in-out, ease-in-out, ease');
+          } catch(e) {
+            // Firefox
+            expect(drawerStyle['transition-timing-function']).to.be.equal('cubic-bezier(0.42, 0, 0.58, 1), cubic-bezier(0.42, 0, 0.58, 1), cubic-bezier(0.25, 0.1, 0.25, 1)');
+          }
+
+          done();
+        });
+      });
+
+      test('drawer uses timing function from property', function(done) {
+        var f, drawer;
+        f = fixture('custom-timing-function-drawer');
+        drawer = f.querySelector('#drawer');
+
+        Polymer.Base.async(function() {
+          var drawerStyle = window.getComputedStyle(drawer);
+
+          try {
```

Comment:
Let's have the browser figure this out.  You can have a fixture, e.g. a div that has `style="transition-timing-function: linear;` so you compare against that parsed CSS value. In that way, we can avoid all these try/catch blocks.